package/MDAnalysis/coordinates/base.py

```diff
@@ -996,12 +1004,30 @@ def add_auxiliary(self, auxname, auxdata, format=None, **kwargs):
 
         The representative value(s) of the auxiliary data for each timestep (as
         calculated by the :class:`~MDAnalysis.auxiliary.base.AuxReader`) are
-        stored in the current timestep in the ``ts.aux`` namespace under *auxname*;
-        e.g. to add additional pull force data stored in pull-force.xvg::
+        stored in the current timestep in the ``ts.aux`` namespace under
+        *aux_spec*; e.g. to add additional pull force data stored in
+        pull-force.xvg::
 
             u = MDAnalysis.Universe(PDB, XTC)
             u.trajectory.add_auxiliary('pull', 'pull-force.xvg')
 
+
+        In the more general case (for example for energy readers),
+        *aux_spec* is expected to be a dictionary that maps the desired
+        attribute name in the ``ts.aux`` namespace to the precise data to be
+        added as identified by a :attr:`data_selector`::
+
+            term_dict = {"temp": "Temperature", "epot": "Potential"}
+            u.trajectory.add_auxiliary(term_dict, "ener.edr")
+
+
+        All data that is present in the (energy) file can be added when the
+        `aux_spec` is omitted. `auxdata` then needs to be passed explicitly
+        like so::
+
+            u.trajectory.add_auxiliary(auxdata="ener.edr")
+
+
```

Comment:
Move to after the simple example with pull force. First explain the simple case to the user, then the complicated one. A short example for getting something out of the energy reader would be good.